syntax = "proto2";

option java_multiple_files = true;
option java_outer_classname = "ProtoIntegerStringDictionaryWrapper";

package org.openstreetmap.atlas.proto;

message ProtoIntegerStringDictionary {
    optional string name = 1;
    repeated string words = 2;
}
